msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -7,49 +7,79 @@ ms.localizationpriority: high
 
 # Create deep links
 
-You can create links to information and features within Teams. The scenarios where creating deep links are useful are as follows:
+Deep links are a navigation mechanism that you can use to connect users with information and features within Teams and Teams apps. Some scenarios where creating deep links can be useful are as follows:
 
 * Navigating the user to the content within one of your app's tabs. For instance, your app can have a bot that sends messages notifying the user of an important activity. When the user taps on the notification, the deep link navigates to the tab so that the user can view more details about the activity.
-* Your app automates or simplifies certain user tasks, such as creating a chat or scheduling a meeting, by pre populating the deep links with required parameters. This avoids the need for users to manually enter information.
+* Your app automates or simplifies certain user tasks, such as creating a chat or scheduling a meeting, by pre-populating the deep links with required parameters. This avoids the need for users to manually enter information.
+
+[!INCLUDE [sdk-include](~/includes/sdk-include.md)]
 
 > [!NOTE]
 >
-> A deeplink launches the browser first before navigating to content. The behaviour of deep links on Teams entities are as follows:
+> The behavior of deep links is dependent on a number of factors. The following list outlines the behavior of deep links on Teams entities.
 >
 > **Tab**:  
-> ✔ Directly navigates to the deeplink url.
+> ✔ Directly navigates to the deep link url.
 >
 > **Bot**:  
-> ✔ Deeplink in card body: Opens in browser first.  
-> ✔ Deeplink added to OpenURL action in Adaptive Card: Directly navigates to the deeplink url.  
+> ✔ Deep link in card body: Opens in browser first.  
+> ✔ Deep link added to OpenURL action in Adaptive Card: Directly navigates to the deep link url.  
 > ✔ Hyperlink markdown text in the card: Opens in browser first.  
 >
 > **Chat**:  
-> ✔ Text message hyperlink markdown: Directly navigates to deeplink url.  
-> ✔ Link pasted in general chat conversation: Directly navigates to deeplink url.
+> ✔ Text message hyperlink markdown: Directly navigates to deep link url.  
+> ✔ Link pasted in general chat conversation: Directly navigates to deep link url.
+>
+> For Teams apps extended across M365 (Outlook/Office) behavior is dependent upon where the deep link is pointing relative to where the Teams app is running. If the Teams app is running within the host the deep link is pointing to it will open directly within the host. If the Teams app is running in a host that differs from where the link is pointing, it will open first in the browser.
```

Comment:
Let's expand all instances of M365 to "Microsoft 365" (to appease Acrolinx branding guidance)